### ISSUE TITLE
-simplify resolution option error 

### ISSUE THREAD

**Issue by jimvermillion** [mbloch/mapshaper#221]
Hi, I'm getting the error:
`[simplify] Error: Missing a simplification amount`

when I run:
```
$(npm bin)/mapshaper \
  -i input_file.shp combine-files snap encoding=utf8 \
  -simplify resolution=25600x14400 keep-shapes \
  -o force world.json format=geojson
```

This still works with versions <= v0.4.49
Have the requirements for the `resolution` option changed?

Thanks!

**Comment by mbloch** [comment]
I released a new version, v0.4.55, that fixes `resolution`. This option can be used with either a single number or WxH. A recent updated accidentally broke parsing of the WxH format. I've added new tests to make sure that this doesn't happen again.

Thanks for reporting this bug :)

**Comment by jimvermillion** [comment]
Awesome, thanks!!
